misc/services/mobile-webui/mobile-webui-frontend/src/containers/activities/picking/PickStepButton.jsx

```diff
@@ -27,11 +29,13 @@ const PickStepButton = ({
 
   const isAlternative = !!altStepId;
   const completeStatus = computePickFromStatus(pickFrom);
+  const catchWeight = catchWeightUOM && parseQRCodeString(pickFrom.huQRCode.code)?.weightNet;
+  const captionToUse = (catchWeight ? catchWeight + catchWeightUOM : undefined) || pickFrom.locatorName;
```

Comment:
why not using formatQtyToHumanReadableStr?

Reply:
checking that out 👀